test/lit.cfg

```diff
@@ -1918,6 +1918,21 @@ if config.lldb_build_root != "":
 # variable.
 config.environment[TARGET_ENV_PREFIX + 'SWIFT_DETERMINISTIC_HASHING'] = '1'
 
+# num_extra_inhabitants on 64 bits differs between mac and everywhere else.
+# Add it, as well as the the 3 previous values, as variables,
+# in order to substitute them in the tests
+if platform.system() == 'Darwin' and (run_os == 'macosx' or run_os == 'darwin'):
+    num_extra_inhabitants_64bit = 2147483647
+elif platform.system() == 'Linux':
+    num_extra_inhabitants_64bit = 4096
+config.substitutions.append( ('%num_extra_inhabitants_64bit', str(num_extra_inhabitants_64bit)))
+complete = "-Dnum_extra_inhabitants_64bit={} ".format(num_extra_inhabitants_64bit)
+
+for i, num in enumerate(range(num_extra_inhabitants_64bit - 1, num_extra_inhabitants_64bit - 1 - 3, -1)):
+    complete += "-Dnum_extra_inhabitants_64bit_minus_{0}={1} ".format(i + 1, num)
+
+config.substitutions.append( ('%add_num_extra_inhabitants', complete))
```

Comment:
Could you postpone this change to a follow up? It doesn't appear to be used in this patch.